tests/org.eclipse.papyrus.uml.diagram.sequence.runtime.tests/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/policies/tests/MessageMoveUITest.java

```diff
@@ -72,6 +76,18 @@ public MessageMoveUITest(boolean rightToLeft, String direction, boolean moveDown
 		this.moveDown = moveDown;
 	}
 
+	@BeforeClass
+	public static void turnOffAutoCreationOfExecutions() {
```

Comment:
We'll probably need this in a bunch of test classes.  Should it be encapsulated in a reusable `Rule`?